data-pipeline/terraform/container_app/main.tf

```diff
@@ -1,5 +1,5 @@
 resource "azurerm_container_app" "data-pipeline" {
-  name                         = "${var.environment-prefix}-ebis-data-pipeline-${var.container-app-name-suffix}"
```

Comment:
How close was the original to the `32` limit? Is the `suffix` going to be unduly truncated and/or should a different naming convention be defined?

Reply:
`s198d01-ebis-data-pipeline-default` was being truncated to `s198d01-ebis-data-pipeline-defau` so it's only a couple of characters; `custom` would be less so.

I've dropped the `data` part which should avoid truncation in this instance. I've added the `substr()` call "just in case".